examples/nextjs-spa/package.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "nextjs-spa",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@ory/client": "^0.2.0-alpha.60",
```

```suggestion
    "@ory/client": "^1.0.0",
```

I believe there is a new version out. This will require some refactoring on the sdk methods since they were changed.

---

Check this PR on how to change it:
https://github.com/ory/elements/pull/56